arrow/src/main/java/org/apache/calcite/adapter/arrow/ArrowRules.java

```diff
@@ -0,0 +1,175 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to you under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.calcite.adapter.arrow;
+
+import org.apache.calcite.adapter.enumerable.EnumerableConvention;
+import org.apache.calcite.plan.Convention;
+import org.apache.calcite.plan.RelOptRule;
+import org.apache.calcite.plan.RelOptRuleCall;
+import org.apache.calcite.plan.RelRule;
+import org.apache.calcite.plan.RelTraitSet;
+import org.apache.calcite.rel.RelNode;
+import org.apache.calcite.rel.convert.ConverterRule;
+import org.apache.calcite.rel.core.Filter;
+import org.apache.calcite.rel.core.Project;
+import org.apache.calcite.rel.logical.LogicalFilter;
+import org.apache.calcite.rel.logical.LogicalProject;
+import org.apache.calcite.rel.type.RelDataType;
+import org.apache.calcite.sql.validate.SqlValidatorUtil;
+
+import com.google.common.collect.ImmutableList;
+
+import org.checkerframework.checker.nullness.qual.Nullable;
+import org.immutables.value.Value;
+
+import java.util.List;
+
+/** Planner rules relating to the Arrow adapter. */
+public class ArrowRules {
+  private ArrowRules() {}
+
+  /** Rule that matches a {@link org.apache.calcite.rel.core.Project} on
+   * an {@link ArrowTableScan} and pushes down projects if possible. */
+  public static final ArrowProjectRule PROJECT_SCAN =
+      ArrowProjectRule.DEFAULT_CONFIG.toRule(ArrowProjectRule.class);
+
+  public static final ArrowFilterRule FILTER_SCAN =
+      ArrowFilterRule.Config.DEFAULT.toRule();
+
+  public static final ArrowToEnumerableConverterRule TO_ENUMERABLE =
+      ArrowToEnumerableConverterRule.DEFAULT_CONFIG
+          .toRule(ArrowToEnumerableConverterRule.class);
+
+  public static final List<RelOptRule> RULES =
+      ImmutableList.of(PROJECT_SCAN,
+          FILTER_SCAN);
```

Comment:
I am sorry to insist with this kind of changes, but IMO all these un-needed newlines make the code less readable, the amount of lines that can fit into the screen at the same time helps, if we waste many lines in this way, we end up scrolling a lot for not much